ignitionrobotics/ign-gazebo
# Adds new RFComms system

# 🎉 New feature

Requires

* https://github.com/ignitionrobotics/ign-gazebo/pull/1416

## Summary


This PR adds a radio frequency comms model atop comms infrastructure created in #1416.

## Test it


* Run the tests.
* Run the example:

1. Launch Gazebo:
```
ign gazebo -v 4 <your_ws>/src/ign-gazebo/examples/worlds/rf_comms.sdf
```
2. Launch a subscriber:
```
ign topic -e -t addr2/rx
```
3. Launch a publisher:
```
cd <your_ws>/src/ign-gazebo/examples/standalone/comms
mkdir build && cd build
cmake ..
make
./publisher addr2
```
You should observe messages on the subscriber terminal.

CC @arjo129 

## Checklist

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸
